www/directives/player/tracks.html

```diff
@@ -1,28 +1,30 @@
-<div>
-	<h3>Overall progress</h3>
-	<table class='table'>
-		<tr ng-repeat='status in tracksCtrl.tracksStatus'>
-			<th width='30%'>
-				<a href='/tracks/{{status.track._id}}'>
-					{{status.track.title}}
-				</a>
-			</th>
-			<td width='3%'>
+<div class='panel panel-default' ng-repeat='status in tracksCtrl.tracksStatus'>
+	<div class='panel-middle' style='vertical-align: top'>
+		<h4 class='title'>
+			<a href='/tracks/{{status.track._id}}'>{{status.track.title}}</a>
+		</h4>
+		<span class='text-muted' ng-bind-html='status.track.desc'></span>
```

Comment:
Yeah, I'm not sure of what we need
* The short one is useful to give an overview of the track
* Why do we need a full one?